src/app/page.tsx

```diff
@@ -1,41 +1,37 @@
-import {
-	EstimatedPackage,
-	PackageOwnership,
-	tideliftMeUp,
-} from "tidelift-me-up";
-
 import { Footer } from "~/components/Footer";
 import { MainArea } from "~/components/MainArea";
 import { OptionsForm } from "~/components/OptionsForm";
 import { ResultDisplay } from "~/components/ResultDisplay";
 import { ScrollButton } from "~/components/ScrollButton";
+import { SearchParamsType, fetchData } from "~/utils/fetchData";
 
+import { metadata as defaultMetadata } from "./layout";
```

Comment:
[Non-Actionable] It would be nice if this had an image... but that's definitely out of scope for this PR 😄. Probably a lot more work. I don't even know what that image would look like.

Opinion: someone should file a feature request issue after this suggesting adding image(s) to the metadata. If you want to, awesome - but I can if you don't have time. 🙂

Reply:
Great suggestion! I went ahead and created a [PR](https://github.com/JoshuaKGoldberg/tidelift-me-up-site/issues/843) 😄

Reply:
🙌 thanks!